pkg/cmd/grafana-cli/commands/install_command.go

```diff
@@ -19,7 +19,13 @@ import (
 	"github.com/grafana/grafana/pkg/plugins/storage"
 )
 
-func validateInput(c utils.CommandLine, pluginFolder string) error {
+const installArgsSize = 2
+
+func validateInput(c utils.CommandLine) error {
+	if c.Args().Len() > installArgsSize {
+		return errors.New("install only supports 2 arguments: plugin and version")
```

Comment:
Maybe before we return the error here, we should add a log with something more descriptive to hopefully help the user form the right command.

For example:

```
./bin/darwin-amd64/grafana cli plugins install autohome-comparequeries-datasource --debug --pluginUrl /Users/wbrowne/Downloads/autohome-comparequeries-datasource.zip                                           
Please specify the correct format. For example ./grafana cli (<command arguments>) plugins install <plugin ID> (<plugin version>)

Error: ✗ install only supports 2 arguments: plugin and version
```

```suggestion
		logger.Info(color.RedString("Please specify the correct format. For example ./grafana cli (<command arguments>) plugins install <plugin ID> (<plugin version>)\n\n"))
		return errors.New("install only supports 2 arguments: plugin and version")
```


Maybe @sympatheticmoose/@josmperez can weigh in here on best lingo

Reply:
For reference, this would look like:
![image](https://github.com/grafana/grafana/assets/1173299/adb03816-ddd3-4ac5-92fd-e24c1f4c801c)